widgets/init.lua

```diff
@@ -13,7 +13,7 @@ _M.awesomemenu = {
    {'manual', apps.manual_cmd},
    {'edit config', apps.editor_cmd .. ' ' .. awesome.conffile},
    {'restart', awesome.restart},
-   {'quit', awesome.quit},
+   {'quit', function() awesome.quit() end},
```

Comment:
Although it's not necessary to create a wrapper function that just calls another function and we can just pass in the original quit function, i'll merge this to be more consistent with the original rc.lua